definition-map.js

```diff
@@ -140,6 +140,26 @@ const QUERIES = {
             isPreview: "Requests the preview URL to launch SCORM content. This requires the current user to be in admin or manager role.",
             type: "The type of the Topic Or Course."
         }
+    },
+    CourseById: {
+        metadescription: "Returns various information about the course and the objects that are contained inside of the course, such as the sections, lessons, and pages associated with the course.",
+        args: {
+            id: "The ID of the course. Passing the ID of the CourseGroup will not work here. You can get the course ID from the Catalog Content query with the displayCourse field."
+        }
+    },
+    UserContentItems: {
+        metadescription: "Returns an array of content that the current user logged in has access to.",
+        args: {
+            kind: "The kind of content you want to fetch",
+            query: "A query to filter the items. For example, if you only wanted to fetch videos you could pass in 'contentType:Video'. A full list of possible queries can be found here: https://support.thoughtindustries.com/hc/en-us/articles/360046307253-Writing-a-Search-Query",
+            sort: "An argument to sort. An example would be 'descending'"
+        }
+    },
+    Pages: {
+        metadescription: "Returns an array of Page objects that are found within courses, such as quiz pages, text pages, video pages, and more.",
+        args: {
+            identifiers: "The IDs of the Page objects. These are available under the CourseById query."
```

Comment:
Same as earlier comment specifying which query these IDs can be found in. Because of the nature of GraphQL, this property could be fetched from any query that returns a `Course` objectType.

Reply:
Would you like me to omit where you can find these then? And just leave it as "The IDs of the Page objects."?

Reply:
the ids of the pages objects isn't descriptive enough imo, can we say it's available under Course > Topics > Lessons > Pages > Ids

Reply:
That feels like the kind of information that would be better suited for the Developer Guide (as an example or how-to) than it would as API documentation. Think `The IDs of the Page objects requested` would be descriptive enough for the same reasons as mentioned in my previous comment.